pom.xml

```diff
@@ -23,7 +23,6 @@
   </licenses>
 
   <modules>
-    <module>annotation</module>
     <module>plugin</module>
```

Comment:
May as well just move the plugin to top level.

Not sure what will happen with https://github.com/jenkinsci/plugin-compat-tester/blob/08e76c5c1f96dd9650962bbee6bbf0e8e52d509b/plugins-compat-tester/src/main/java/org/jenkins/tools/test/hook/StructsHook.java#L41-L52

Reply:
> May as well just move the plugin to top level.

Doing this in this PR would obscure the basic point of the change by moving a lot of files around. I think this can be taken care of in subsequent PRs.

Reply:
(Note that GitHub will properly display file renames.)